app/src/main/res/layout/screen_patient_search.xml

```diff
@@ -1,65 +1,150 @@
-    android:elevation="@dimen/elevation_toolbar"
-    android:gravity="center_vertical"
-    tools:ignore="UnusedAttribute">
-
-    <ImageButton
-      android:id="@+id/patientsearch_back"
-      android:layout_width="wrap_content"
-      android:layout_height="wrap_content"
-      android:background="?attr/selectableItemBackgroundBorderless"
-      android:contentDescription="@string/patientsearch_go_back"
-      android:padding="@dimen/spacing_12"
-      android:src="@drawable/ic_arrow_back_24dp"
-      android:tint="@color/black_opacity_50" />
-
-    <!-- privateImeOptions="nm" is used by the AOSP and some keyboards
-         to hide the suggestions bar entirely. Gboard ignores this
-         unfortunately. -->
-    <EditText
-      android:id="@+id/patientsearch_fullname"
-      android:layout_width="0dp"
-      android:layout_height="wrap_content"
-      android:layout_weight="1"
-      android:background="@null"
-      android:hint="@string/patientsearch_name_or_phone"
-      android:imeOptions="actionSearch"
-      android:inputType="text|textCapWords|textFilter"
-      android:privateImeOptions="nm">
-
-      <requestFocus />
-    </EditText>
-  </LinearLayout>
-
-  <Button
-    android:id="@+id/patientsearch_new_patient"
-    style="@style/Clinic.Button.PatientEntry"
-    android:layout_width="match_parent"
+  <ImageButton
+    android:id="@+id/patientsearch_back"
+    android:layout_width="wrap_content"
     android:layout_height="wrap_content"
-    android:layout_below="@id/patientsearch_toolbar"
-    android:contentDescription="@string/patientsearch_new_patient"
-    android:drawableStart="@drawable/ic_add_circle_outline_24dp"
-    android:text="@string/patientsearch_new_patient"
-    android:visibility="gone" />
-
-  <android.support.v7.widget.RecyclerView
-    android:id="@+id/patientsearch_recyclerview"
+    android:background="?attr/selectableItemBackgroundBorderless"
+    android:padding="@dimen/spacing_16"
+    android:src="@drawable/ic_arrow_back_24dp"
+    android:tint="@color/gray_600"
+    tools:ignore="ContentDescription" />
+
+  <ScrollView
     android:layout_width="match_parent"
     android:layout_height="match_parent"
     android:layout_above="@+id/patientsearch_search"
-    android:layout_below="@id/patientsearch_new_patient"
-    tools:listitem="@layout/list_patient_search" />
+    android:layout_toEndOf="@+id/patientsearch_back"
+    android:scrollbarStyle="insideOverlay"
+    android:scrollbars="vertical">
+
+    <LinearLayout
+      android:layout_width="match_parent"
+      android:layout_height="wrap_content"
+      android:orientation="vertical"
+      android:paddingBottom="64dp"
+      android:paddingEnd="@dimen/spacing_8"
+      android:paddingTop="@dimen/spacing_16"
+      tools:ignore="RtlSymmetry">
+
+      <FrameLayout
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content">
+
+        <android.support.design.widget.TextInputLayout
```

Comment:
`TextInputLayout` is a sub-class of `LinearLayout`. Can't we add the clear button as a child of the `TextInputLayout` itself?

Reply:
No, because the clear button needs to appear in front of the text field